src/main/java/com/onaio/steps/Properties.java

```diff
@@ -36,11 +36,11 @@
     private static final Map<String, String> props = new HashMap<String, String>();
 
     static {
-        props.put(ENDPOINT_URL, "http://steps.ona.io/upload-file");
+        props.put(ENDPOINT_URL, "http://stepsv2.ona.io/upload-file");
         props.put(HH_FORM_ID,"STEPS_Instrument_V3_1");
         props.put(HH_MIN_AGE, "18");
         props.put(HH_MAX_AGE, "69");
-        props.put(IMPORT_URL, "http://steps.ona.io/export");
+        props.put(IMPORT_URL, "http://stepsv2.ona.io/export");
```

Comment:
It's configurable in the settings. The values that were changed are the default values